SalesforceCommerceCloud/pwa-kit
# W-14308527 Pricing display on cart, checkout, wishlist pages

# Description
Pricing display on cart checkout wishlist pages due to product tile revampe



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Use DisplayPrice component in item-variant component
- Allow DisplayPrice to take quantity into account
- 
# How to Test-Drive This PR

- Checkout code
- Run retail app
- Go to PDP, add a product
- Confirm that pricing in AddToCart Modal is correct
- Confirm that cart page is showing pricing correctly with strikethrough price
- Add an item to wishlist
- Confirm that pricing is displaying according to the quantity in wishlist page

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
